### PR TITLE
POM pattern implemented to test the flow of buying a t-shirt

### PR DESCRIPTION
**Pull request by danielgalvis98**

### CODE REVIEW

File: src/page/index.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export { MenuContentPage } from './menu-content.page';
+export { ProductListPage } from './product-list.page';
+export { ProductAddedModalPage } from './product-added-modal.page';
+export { OrderSummaryPage } from './order-summary.page';
+export { SignInStepPage } from './sign-in-step.page';
+export { AddresStepPage } from './address-step.page';
+export { ShippingStepPage } from './shipping-step.page';
+export { PaymentStepPage } from './payment-step.page';
+export { BankPaymentPage } from './bank-payment.page';
```
Comment (holgiosalos) [inline src/page/index.ts:9]: You are missing `summary-step.page.ts` and confusing that page with the `order-summary.page.ts`, which should contain the last step of the buying process.

File: test/buy-tshirt.spec.ts [hunk 2]
```diff
@@ -1,38 +1,48 @@
...
+    await menuContentPage.goToTshirtMenu();
     await(browser.sleep(3000));
-    await $('#center_column a.button.ajax_add_to_cart_button.btn.btn-default').click();
+    await productListPage.addShirtToCart();
     await(browser.sleep(3000));
-    await $('[style*="display: block;"] .button-container > a').click();
+    await productAddedModalPage.proccedToCheckout();
     await(browser.sleep(3000));
-    await $('.cart_navigation span').click();
+    await orderSummaryPage.confirmAndCheckout();
```
Comment (holgiosalos) [inline test/buy-tshirt.spec.ts]: here, you should use the page object that should implement the `summary-step.page.ts`

File: test/buy-tshirt.spec.ts [hunk 3]
```diff
@@ -1,38 +1,48 @@
...
+    await shippingStepPage.goToPaymentMenu();
     await(browser.sleep(3000));
-    await $('#HOOK_PAYMENT > div:nth-child(1) > div > p > a').click();
+    await paymentStepPage.payByBankwire();
     await(browser.sleep(3000));
-    await $('#cart_navigation > button > span').click();
+    await bankPaymentPage.confirmOrder();
     await(browser.sleep(3000));
 
     await expect($('#center_column > div > p > strong').getText())
```
Comment (holgiosalos) [inline test/buy-tshirt.spec.ts]: here you should implement the page object corresponding to the `order-summary.page.ts`. **Do not** move the expectation to that page, just the act of getting the title of the final step.

File: src/page/index.ts (hunk 1 above)
Comment (danielgalvis98) [reply src/page/index.ts:9]: Yes, I missed it, thanks for the advice.
I think it is fixed now :)

**Review by holgiosalos — APPROVED** [review]
Please, in your local, run the linter with the --fix option, if any file is updated, update the PR with those changes

**Comment by danielgalvis98** [comment]
No file was updated runing linter with the --fix option

**Pull request merged**